Default upcoming changes to false, and do not look for it

```
freecodecamp@stg-vm-api-alpha:~/freeCodeCamp$ npm run ensure-env && npm run build:server

> @freecodecamp/freecodecamp@0.0.1 ensure-env /home/freecodecamp/freeCodeCamp
> cross-env DEBUG=fcc:* node ./tools/scripts/build/ensure-env.js

/home/freecodecamp/freeCodeCamp/tools/scripts/build/ensure-env.js:45
      throw Error(`Env. variable ${key} is missing, build cannot continue`);
      ^

Error: Env. variable showUpcomingChanges is missing, build cannot continue
    at Object.<anonymous> (/home/freecodecamp/freeCodeCamp/tools/scripts/build/ensure-env.js:45:13)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/freecodecamp@0.0.1 ensure-env: `cross-env DEBUG=fcc:* node ./tools/scripts/build/ensure-env.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @freecodecamp/freecodecamp@0.0.1 ensure-env script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/freecodecamp/.npm/_logs/2020-09-24T16_38_54_905Z-debug.log
```

--- a/config/env.js
+++ b/config/env.js
@@ -39,16 +39,16 @@ module.exports = Object.assign(locations, {
       ? null
       : stripePublicKey,
   algoliaAppId:
-    !algoliaAppId || algoliaAppId === 'Algolia app id from dashboard'
+    !algoliaAppId || algoliaAppId === 'app_id_from_algolia_dashboard'
       ? null
       : algoliaAppId,
   algoliaAPIKey:
-    !algoliaAPIKey || algoliaAPIKey === 'Algolia api key from dashboard'
+    !algoliaAPIKey || algoliaAPIKey === 'api_key_from_algolia_dashboard'
       ? null
       : algoliaAPIKey,
   paypalClientId:
     !paypalClientId || paypalClientId === 'id_from_paypal_dashboard'
       ? null
       : paypalClientId,
-  showUpcomingChanges: showUpcomingChanges && showUpcomingChanges === 'true'
+  showUpcomingChanges: showUpcomingChanges === 'true'
 });

--- a/tools/scripts/build/ensure-env.js
+++ b/tools/scripts/build/ensure-env.js
@@ -45,6 +45,12 @@ if (FREECODECAMP_NODE_ENV === 'production') {
       throw Error(`Env. variable ${key} is missing, build cannot continue`);
     }
   }
+
+  if (env['environment'] !== 'production')
+    throw Error("Production environment should be 'production' ");
+
+  if (env['showUpcomingChanges'])
+    throw Error("SHOW_UPCOMING_CHANGES should never be 'true' in production");
 }
 
 fs.writeFileSync(`${clientPath}/config/env.json`, JSON.stringify(env));

